package.json

```diff
@@ -62,7 +62,7 @@
     "error-stack-parser": "^2.0.6",
     "find-up": "^5.0.0",
     "html-entities": "^2.1.0",
-    "loader-utils": "^2.0.4",
+    "loader-utils": "^3.2.1",
```

Comment:
Do we still consume `loader-utils` directly? If not I'm leaning towards removing this as a dependency to let downstream decide on what version is best for them.